src/web/src/components/Posts/Timeline.tsx

```diff
@@ -49,7 +51,9 @@ const Timeline = ({ pages, nextPage }: Props) => {
   // Iterate over all the pages (an array of arrays) and then convert all post
   // elements to <Post>
   const postsTimeline = pages.map((page: Post[]): Array<ReactElement> | ReactElement =>
-    page.map(({ id, url }: Post) => <PostComponent postUrl={url} key={id} />)
+    page.map(({ id, url }: Post) => (
+      <PostComponent postUrl={url} key={id} currentPost={++currentCount} totalPosts={totalPosts} />
```

Comment:
ESLint doesn't allow for unary operators. Instead of  `++currentCount` use `currentCount += 1`